### ISSUE TITLE
[Beta 7] On essentia pipes balance.

### ISSUE THREAD

**Issue by zabodan** [Azanor/thaumcraft-beta#391]
I love essentia pipes. Golems are cool, but nothing beats large pipe-based system for generating, sorting, storing and using essentia.

Compared with TC4 essentia amounts went up, jars became 4x larger, smeltery throughput went up at least 3x (thaumium smeltery + pair of aux slurry pumps), but generic pipes, buffers and centrifuges still operate at old speed.

Buffer chain attached to thaumium smeltery with pair of pumps and three alembics (one main, plus one for each pump) can not extract essentia fast enough -- it only manages to draw from two alembics. If you add feedback loops from centrifuges and a recycling line from the thaumatorium, your main buffer chain clogs very quickly -- essentia just doesn't flow fast enough.

I'd suggest to increase troughput of basic pipes at least 4x, increase essentia buffers size 2-4x, and maybe make centrifuges a bit faster. If pipes are faster it will be possible to scale up by building more centrifuges, but early game, when you have only one centrifuge, it is painful to watch it do its thing.

On a separate topic, is arcane trapdoor coming back? Infernal furnace just isn't the same with usual trapdoor.

**Comment by zabodan** [comment]
New knowledge aquired, therefore a correction is necessary: Emptying Essentia Transfuser largely trivializes thaumatorium setup. One transfuser is enough to feed thaumatorium with exact amount of essentia from nearby jars, without any pipes. Which means less pipes are needed overall. Still, pipes are slow ;-)

Oh, and by arcane trapdoor I meant item grate.

**Comment by zabodan** [comment]
It takes time and experimentation to really feel the shift in balance between TC4 and TC6.

Focusing on clear essentia generation (minimizing flux per produced essentia) seems much more important, than on energy efficiency (maximizing produced essentia per alumentum). Smeltery with a pair of slurry pumps generates flux too quickly -- it's fun to try once, but in the long run it pollutes environment too much.

Transfusers together with essentia mirrors make essentia transportation very easy (from smelteries to storage, from storage to thaumatorium and golem press). One buffer chain around the storage room is enough to sort all incoming essentia and centrifuge excess of compounds. After moving smeltery to a different chunk and redesigning essentia storage I haven't seen any essentia jams.

To summarize, pipes are fine ;-)

**Comment by Azanor** [comment]
This issue was moved to Azanor/thaumcraft-suggestions#242
